packages/js/onboarding/src/components/WooOnboardingTask/WooOnboardingTask.tsx

```diff
@@ -35,18 +35,29 @@ export const trackView = async ( taskId, variant ) => {
 	} );
 };
 
-let experimentalVariant;
+let experimentalVariant: string | undefined;
+type WooOnboardingTaskProps = {
+	id: string;
+	variant?: string;
+};
+
+type WooOnboardingTaskSlotProps = Slot.Props & {
+	id: string;
+};
+
 /**
  * A Fill for adding Onboarding tasks.
  *
  * @slotFill WooOnboardingTask
  * @scope woocommerce-tasks
- * @param {Object} props          React props.
- * @param {string} props.variant  The variant of the task.
- * @param {Object} props.children React component children
- * @param {string} props.id       Task id.
+ * @param {Object} props           React props.
+ * @param {string} [props.variant] The variant of the task.
```

Comment:
I don't understand the `[]` add here, @chihsuan. Can you shed a light? Thanks!

Reply:
It indicates that `variant` is optional.

https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
TIL. Thanks!